### PR TITLE
{bio}[goolf/1.4.10] blasr smrtanalysis-2.1 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
wrapper PR for #699 by @pforai

### CODE REVIEW

File: easybuild/easyconfigs/b/blasr/blasr-smrtanalysis-2.1-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+#   Authors: Petar <petar.forai@gmi.oeaw.ac.at> Forai
+# ----------------------------------------------------------------------------
+
+easyblock = 'MakeCp'
+
+name = 'blasr'
+version = 'smrtanalysis-2.1'
+
+homepage = 'https://github.com/PacificBiosciences/blasr'
+description = """BLASR (Basic Local Alignment with Successive Refinement) rapidly maps reads to genomes by finding the highest scoring local alignment or set of local alignments between the read and the genome. Optimized for PacBio's extraordinarily long reads and taking advantage of rich quality values, BLASR maps reads rapidly with high accuracy."""
```
Comment (ocaisa) [inline easybuild/easyconfigs/b/blasr/blasr-smrtanalysis-2.1-goolf-1.4.10.eb]: This line is too long...120 char limit

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/42f363fb7d6b68db56ea for a full test report.

File: easybuild/easyconfigs/b/blasr/blasr-smrtanalysis-2.1-goolf-1.4.10.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/b/blasr/blasr-smrtanalysis-2.1-goolf-1.4.10.eb]: sorry, wasn't aware of that ;-)

Fixed.

**Comment by rjeschmi** [comment]
Test report by @rjeschmi
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux centos 6.6, AMD Opteron(tm) Processor 6380, Python 2.7.8
See https://gist.github.com/27de6119722b8c1b3b50 for a full test report.

**Comment by rjeschmi** [comment]
Test report by @rjeschmi
**FAILED**
Build succeeded for 0 out of 12 (1 easyconfigs in this PR)
Linux centos 6.6, AMD Opteron(tm) Processor 6380, Python 2.7.8
See https://gist.github.com/9dbaf8ae650dec127acb for a full test report.

**Comment by rjeschmi** [comment]
Test report by @rjeschmi
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in this PR)
Linux centos 6.6, AMD Opteron(tm) Processor 6380, Python 2.6.6
See https://gist.github.com/33b72c587bdd4e9d4fe8 for a full test report.

**Comment by rjeschmi** [comment]
@boegel Seems fine (failures were my fault).

File: easybuild/easyconfigs/b/blasr/blasr-smrtanalysis-2.1-goolf-1.4.10.eb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+name = 'blasr'
+version = 'smrtanalysis-2.1'
+
+homepage = 'https://github.com/PacificBiosciences/blasr'
+description = """BLASR (Basic Local Alignment with Successive Refinement) rapidly maps reads to genomes by finding the
+ highest scoring local alignment or set of local alignments between the read and the genome. Optimized for PacBio's
+ extraordinarily long reads and taking advantage of rich quality values, BLASR maps reads rapidly with high accuracy."""
+
+
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/b/blasr/blasr-smrtanalysis-2.1-goolf-1.4.10.eb]: remove extra line

File: easybuild/easyconfigs/b/blasr/blasr-smrtanalysis-2.1-goolf-1.4.10.eb [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+buildopts = 'HDF5INCLUDEDIR=${EBROOTHDF5}/include HDF5LIBDIR=${EBROOTHDF5}/lib'
+
+files_to_copy = [(['alignment/bin/wordCounter', 'alignment/bin/printReadWordCount', 'alignment/bin/blasr',
+                   'alignment/bin/sdpMatcher', 'alignment/bin/swMatcher', 'alignment/bin/kbandMatcher',
+                   'alignment/bin/sawriter', 'alignment/bin/saquery', 'alignment/bin/samodify',
+                   'alignment/bin/printTupleCountTable', 'alignment/bin/malign',
+                   'alignment/bin/cmpPrintTupleCountTable', 'alignment/bin/removeAdapters',
+                   'alignment/bin/tabulateAlignment', 'alignment/bin/samatcher', 'alignment/bin/sals',
+                   'alignment/bin/saprinter', 'alignment/bin/buildQualityValueProfile', 'alignment/bin/extendAlign',
+                   'alignment/bin/guidedalign', 'alignment/bin/pbmask'],'bin')]
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/b/blasr/blasr-smrtanalysis-2.1-goolf-1.4.10.eb]: turn into 4-space indentation, preferred; trim double empty lines
overall ok.

File: easybuild/easyconfigs/b/blasr/blasr-smrtanalysis-2.1-goolf-1.4.10.eb [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+toolchainopts = {'optarch': True, 'pic': True}
+
+sources = ['%(version)s.tar.gz']
+source_urls = ['https://github.com/PacificBiosciences/blasr/archive/']
+
+patches = ['blasr-%(version)s_non-static.patch']
+
+dependencies = [
+    ('HDF5', '1.8.9'),
+]
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/b/blasr/blasr-smrtanalysis-2.1-goolf-1.4.10.eb]: collapse to 1 line maybe?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/f2d383a112c157129261 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @ocaisa and @fgeorgatos, and for testing @rjeschmi!
And @pforai for kicking this off in #699 ;-)

Going in!

**Pull request merged**